### DISCUSSION TITLE
32 bit OS support

### DISCUSSION THREAD

**Discussion by balternate** [ppy/osu#18209] [Ideas] [answered]
hey can you make a 32 bit build pls?

**Comment by NotGumballer91** [comment]
No.

**Reply by frenzibyte** [reply]
Not even searching first?

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/issues/3984

**Comment by FREINYES** [comment]
> hey can you make a 32 bit build pls?

Download version 2018
